src/components/SettlementButton.js

```diff
@@ -80,15 +80,20 @@ class SettlementButton extends React.Component {
             [CONST.IOU.PAYMENT_TYPE.EXPENSIFY]: {
                 text: this.props.translate('iou.settleExpensify'),
                 icon: Expensicons.Wallet,
-                value: ReportUtils.isExpenseReport(this.props.iouReport) ? CONST.IOU.PAYMENT_TYPE.VBBA : CONST.IOU.PAYMENT_TYPE.EXPENSIFY,
+                value: CONST.IOU.PAYMENT_TYPE.EXPENSIFY,
+            },
+            [CONST.IOU.PAYMENT_TYPE.VBBA]: {
+                text: this.props.translate('iou.settleExpensify'),
+                icon: Expensicons.Wallet,
+                value: CONST.IOU.PAYMENT_TYPE.VBBA,
```

Comment:
What is this doing? And should we add a new icon to distinguish between Wallet and VBBA payments?

Reply:
They will never be shown at the same time, VBBA is only for expense reports and the Wallet is for IOUs so having same icon is fine I think. But we could explore that, maybe easy to add in another PR later.

I just wanted to split these up as it is confusing we use the same button and the key (pyament_type.expensify) but we change the value only based on the type of the report.

Reply:
thanks, that makes sense